ibm_mq/tests/test_ibm_mq_int.py

```diff
@@ -188,15 +188,22 @@ def test_check_all(aggregator, get_check, instance_collect_all, seed_data, dd_ru
     assert_all_metrics(aggregator)
 
 
-def test_check_skip_reset_queue_metrics(aggregator, get_check, instance_collect_all, seed_data, dd_run_check):
-    instance_collect_all['collect_reset_queue_metrics'] = False
+@pytest.mark.parametrize(
+    'collect_reset_queue_metrics',
+    [False, True],
```

Comment:
Out of curiosity, what does the `[False, True]` do here?

Reply:
They are the different values of `collect_reset_queue_metrics` to try. We can then use  `collect_reset_queue_metrics` as a variable in the test: https://docs.pytest.org/en/6.2.x/parametrize.html